src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -9914,8 +9993,6 @@ private static bool IsNullabilityMismatch(TypeSymbol type1, TypeSymbol type2)
 
         public override BoundNode? VisitInterpolatedStringArgumentPlaceholder(BoundInterpolatedStringArgumentPlaceholder node)
         {
-            // These placeholders don't yet follow proper placeholder discipline
-            AssertPlaceholderAllowedWithoutRegistration(node);
             SetNotNullResult(node);
```

Comment:
>SetNotNullResult [](http://example.com/codeflow?start=12&length=16)

VisitPlaceholderWithReplacement?

Reply:
I am unsure how this got reverted, must have been something with the merge?